### ISSUE TITLE
Installation Issue, Ubuntu 22.10   ".txt file does not exist!!"

### ISSUE THREAD

**Issue by poon-holder** [cryinkfly/Autodesk-Fusion-360-for-Linux#307]
**Describe the bug**
Failure to install correctly

**To Reproduce**
Steps to reproduce the behaviour:
Installation via your installer 
" mkdir -p "$HOME/.fusion360/bin" && cd "$HOME/.fusion360/bin" && wget -N https://raw.githubusercontent.com/cryinkfly/Autodesk-Fusion-360-for-Linux/main/files/builds/stable-branch/bin/install.sh && chmod +x install.sh && ./install.sh "

**Expected behavior**
clean install, ability to launch F360 from icon or command line, instead we receive: .txt file does not exist!

![Screenshot from 2023-03-04 12-08-04](https://user-images.githubusercontent.com/93160341/222900399-1200ace7-43a5-4ade-8be3-695a83b48223.png)

**Desktop (please complete the following information):**
- Version number of Autodesk Fusion 360: AFAIA: V.2.0.15299
- Graphics card model + Graphics card driver (Version number) Nvidia RTX A4000, 252 driver
- Operating system Ubuntu 22.10 64bit 
- Desktop environment GNOME 43.1, X11
- DXVK- or OpenGL-Mode: both failed  

**Additional context**
Many thanks for you time, effort and outlook. 
I am but a smooth brain, forgive me. I have had success with your installer on the recent stable version of Ubuntu, however my current GPU fails with 252 driver for that release. I have only had success with the 252 on Ubuntu 22.10 but alas, you most brilliant work around for F360 does not function any longer.

I have had difficulty uninstalling F360 and have now got 2 instances of it.

I take it the most stable distro would be opensuse?
I am happy to jump ship to get this cad "workstation-esque" machine on the road.
Many thanks for you time.
PS where is best to buy you a drink? Do you still use your amazon wish list? SSD's and printer ink etc?
Take care

**Comment by jakehyvonen** [comment]
I'm getting this error as well on my system:

Fusion version: 2.0.15277
GPU: NVIDIA Corporation GA104 [GeForce RTX 3070 Lite Hash Rate] (rev a1)
OS: Ubuntu 22.04.2 LTS
Desktop: Ubuntu/GNOME
DXVK- or OpenGL-Mode: didn't try

But agreed that this is an amazing project and cryinkfly deserves a drink, or at least printer ink!

**Comment by cryinkfly** [comment]
Greetings @poon-holder,

so if you did the standard installation, then you only need to delete the entire content in the folder "$HOME/.fusion360/wineprefixes/default" to start a new attempt.

$HOME = your username (home directory)

You can do this via your file manager (e.g. Thunar) or via the terminal.

> How much money do you have to budget for it to work again, for dummies (noobs) :) (computer illiterate)

That's relatively difficult to assess ... Because it's the case that in the summer I'm on the road a lot because of my main job and therefore have less time than in the winter.

And then I look after other projects in addition to this one.

In short, any support from the community is worth its weight in gold to keep such projects running better and for that reason there are ways to do it.

So if you're really interested in helping me, check out the following link: https://cryinkfly.com/sponsoring/

> I am truly grateful for the work you have put in on top of your busy schedule!

Thanks so much!

**Comment by poon-holder** [comment]
Thank you kindly Mr Zabka for you detailed response.
The fresh install went without a hitch. Un till trying to save bodies as mesh onto local computer.
Here the files are seen in the fusion browser but not to be found in the computer.
this either from '3D print' tab or by right clicking on body and selecting 'save as mesh'.

So,
 your install works well again,
fusion launches, cursor good etc
bodies WILL convert to mesh.
Can not then export or save mesh outside of fusion.

Is this issue #362 ?

Many thanks again.

**Comment by poon-holder** [comment]
Ok, so

my ape mind figured it out, mesh files are being saved to:

/home/*****/.fusion360/wineprefixes/default/drive_c/users/*****/Documents/

where ***** is your username.

Hope this helps others.

Steve i still owe you a drink!
